reference/3.0/Microsoft.PowerShell.Core/About/about_Registry_Provider.md

```diff
@@ -121,21 +126,21 @@ PS C:\ > Get-Item -Path HKLM:\SYSTEM\CurrentControlSet\Services\Spooler
     Hive: HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services
 
 
-Name                           Property
-----                           --------
-Spooler                        DependOnService    : {RPCSS, http}
-                               Description        : @%systemroot%\system32\spoolsv.exe,-2
-                               DisplayName        : @%systemroot%\system32\spoolsv.exe,-1
-                               ErrorControl       : 1
-                               FailureActions     : {16, 14, 0, 0...}
-                               Group              : SpoolerGroup
-                               ImagePath          : C:\WINDOWS\System32\spoolsv.exe
-                               ObjectName         : LocalSystem
-                               RequiredPrivileges : {SeTcbPrivilege, SeImpersonatePrivilege, SeAuditPrivilege,
-                               SeChangeNotifyPrivilege...}
-                               ServiceSidType     : 1
-                               Start              : 2
-                               Type               : 272
+Name        Property
+----        --------
+Spooler     DependOnService    : {RPCSS, http}
+            Description        : @%systemroot%\system32\spoolsv.exe,-2
+            DisplayName        : @%systemroot%\system32\spoolsv.exe,-1
+            ErrorControl       : 1
+            FailureActions     : {16, 14, 0, 0...}
+            Group              : SpoolerGroup
+            ImagePath          : C:\WINDOWS\System32\spoolsv.exe
+            ObjectName         : LocalSystem
+            RequiredPrivileges : {SeTcbPrivilege, SeImpersonatePrivilege,
+            SeAuditPrivilege, SeChangeNotifyPrivilege...}
```

Comment:
Should this line get spaced over so it doesn't look like a new property?

Reply:
Fixed